src/main/java/model/PreferencesModel.java

```diff
@@ -1,8 +1,108 @@
 package model;
 
+import java.io.*;
+
+import observe.Notifier;
 import observe.Observable;
 import observe.PreferencesObserver;
 
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
 public class PreferencesModel extends Observable<PreferencesObserver> {
 
+    private boolean musicEnabled  = true;
+    private boolean soundsEnabled = true;
+
+    private final String PATH = "preferences.xml";
+
+    /**
+     * This variable has to prevent the algorithm to enter an infite loop as during the loading process
+     * a temporal PreferencesModel-instance will be created which would itself call the loading process again.
+     * Which means, the first PreferencesModel sets this variable to 'true' in order to prevent the other Models
+     * from loading the file again.
+     */
+    private static boolean objexists = false;
+    
+    public PreferencesModel() {
+
+        if (!objexists) {
```

Comment:
I can't make a static method. Firstly because the try-catch block would cause trouble as I can't notify the Observers from a static method if an error occurs. Secondly, I can't set the constructor to private, because the Marshaller has to access it.